### PR TITLE
Clean up: content type pattern

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions

MAJOR changes:
none

SPECIAL ATTENTION:
none

### CODE REVIEW

File: schemas/data/contentTypePattern.schema.tpl.json [hunk 1]
```diff
@@ -6,21 +6,21 @@
   ],
   "properties": {
     "contentType": {
-      "_instruction": "Enter the content type that can be defined by the given regular expression for file names/extentions.",
+      "_instruction": "Add the content type that can be defined by the regular expression for this content type pattern (e.g., for file extensions).",
```
Comment (lzehl) [inline schemas/data/contentTypePattern.schema.tpl.json]: ... of this content type pattern (e.g., for file extensions).

File: schemas/data/contentTypePattern.schema.tpl.json [hunk 2]
```diff
@@ -6,21 +6,21 @@
...
-      "_instruction": "Enter a lookup label for this content type pattern."
+      "_instruction": "Enter a lookup label for this content type pattern that may help you to find this instance more easily."
     },
     "regex": {
       "type": "string",
       "_formats": [
         "ECMA262"
       ],
-      "_instruction": "Enter a regular expression for the file names/extensions that defines the give content type."
+      "_instruction": "Enter the regular expression for the file names and/or extensions that defines this content type pattern."
```
Comment (lzehl) [inline schemas/data/contentTypePattern.schema.tpl.json]: "Enter a regular expression for the common part of the names of files (including file path and extension) that are all formatted using the content type defined in this content type pattern." 

(feel free to optimize further)

**Review by lzehl — CHANGES_REQUESTED** [review]

File: schemas/data/contentTypePattern.schema.tpl.json (hunk 1 above)
Comment (UlrikeS91) [reply schemas/data/contentTypePattern.schema.tpl.json]: done

File: schemas/data/contentTypePattern.schema.tpl.json (hunk 2 above)
Comment (UlrikeS91) [reply schemas/data/contentTypePattern.schema.tpl.json]: made another suggestion, let me know if that works

**Review by lzehl — APPROVED** [review]

**Pull request merged**